packages/extension-chakra-storefront/src/components/field/index.jsx

```diff
@@ -76,7 +76,9 @@ const Field = ({
                                         >
                                             <PasswordIcon color="gray.500" boxSize={6} />
                                         </IconButton>
-                                    ) : undefined
+                                    ) : (
+                                        _inputProps?.endElement
```

Comment:
In v3, the pattern is to use `InputGroup` with an `endElement` prop instead of `InputRightElement`.